.github/workflows/ci.yml

```diff
@@ -1,23 +1,39 @@
 name: CI
+
 on:
-  - push
-  - pull_request
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
 jobs:
   test:
+    runs-on: ubuntu-latest
     strategy:
       matrix:
+        python-version: [3.11]
         include:
           - name: Lint
-            command: bin/lint
-    runs-on: ubuntu-latest
+            command: make lint
+          - name: Reformat
+            command: make reformat
```

Comment:
I think we should remove this call since `make reformat` is most often only useful locally. Since black will happily reformat the code in place in CI, this isn't actually checking that code is reformatted.

Perhaps a step in the `bin/lint` to run `black --check` would be best.